sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -0,0 +1,286 @@
+    return V1beta1Experiment(
+        api_version=constants.API_VERSION,
+        kind=constants.EXPERIMENT_KIND,
+        metadata=metadata,
+        spec=V1beta1ExperimentSpec(
+            max_trial_count=3,
+            parallel_trial_count=2,
+            max_failed_trial_count=1,
+            algorithm=algorithm_spec,
+            objective=objective_spec,
+            parameters=parameters,
+            trial_template=trial_template,
+        )
+    )
+
+
+def create_experiment(
+    name: Optional[str] = None,
+    generate_name: Optional[str] = None
+) -> V1beta1Experiment:
+    experiment_namespace = "test"
+
+    if name is not None:
+        metadata = V1ObjectMeta(name=name, namespace=experiment_namespace)
+    elif generate_name is not None:
+        metadata = V1ObjectMeta(generate_name=generate_name, namespace=experiment_namespace)
+    else:
+        metadata = V1ObjectMeta(namespace=experiment_namespace)
+
+    algorithm_spec=V1beta1AlgorithmSpec(
+        algorithm_name="random"
+    )
+
+    objective_spec=V1beta1ObjectiveSpec(
+        type="minimize",
+        goal= 0.001,
+        objective_metric_name="loss",
+    )
+
+    parameters=[
+        V1beta1ParameterSpec(
+            name="lr",
+            parameter_type="double",
+            feasible_space=V1beta1FeasibleSpace(
+                min="0.01",
+                max="0.06"
+            ),
+        ),
+        V1beta1ParameterSpec(
+            name="momentum",
+            parameter_type="double",
+            feasible_space=V1beta1FeasibleSpace(
+                min="0.5",
+                max="0.9"
+            ),
+        ),
+    ]
+
+    trial_template = generate_trial_template()
+
+    experiment = generate_experiment(
+        metadata,
+        algorithm_spec,
+        objective_spec,
+        parameters,
+        trial_template
+    )
+    return experiment
+
+
+test_create_experiment_data = [
+    (
+        "experiment name and generate_name missing",
+        {"experiment": create_experiment()},
+        ValueError,
+    ),
+    (
+        "create_namespaced_custom_object timeout error",
+        {
+            "experiment": create_experiment(name="experiment-mnist-ci-test"),
+            "namespace": "timeout",
+        },
+        TimeoutError,
+    ),
+    (
+        "create_namespaced_custom_object conflict error",
+        {
+            "experiment": create_experiment(name="experiment-mnist-ci-test"),
+            "namespace": "conflict",
+        },
+        Exception,
+    ),
+    (
+        "create_namespaced_custom_object runtime error",
+        {
+            "experiment": create_experiment(name="experiment-mnist-ci-test"),
+            "namespace": "runtime",
+        },
+        RuntimeError,
+    ),
+    (
+        "valid flow with experiment type V1beta1Experiment and name",
+        {
+            "experiment": create_experiment(name="experiment-mnist-ci-test"),
+            "namespace": "test",
+        },
+        "success",
```

Comment:
Can we make this string as constant ?

Reply:
I created a constant to replace `"success"`.